### ISSUE TITLE
The first click in my GIFs are all just focusing the window I want to record

### ISSUE THREAD

**Issue by chriscalo** [Gifox/Tracker#203]
Scenario:
1. I want to record a window
2. Get the window ready
3. Click Gifox, select window recording mode
4. Click the window and then click the record button 🔴
5. Click the window I'm recording to focus it

This results in a distracting click at the start of every full-window GIF I create. Could Gifox not steal focus somehow? Or immediately focus the recorded window as soon as I click record?

**Comment by iby** [comment]
@chriscalo apologies for the lagged reply. This should work without stealing the focus if you use shortcuts, check out Gifox preferences to see what they are.

![image](https://user-images.githubusercontent.com/1086845/131554619-68aedda7-3448-4776-93ee-ea9047a6e24c.png)

Unfortunately there's no good way of making it work with the status bar icon, because it opens the library popover and macOS treats it as a window, which gets activated when gets displayed and deactivates the currently active one.

**Comment by chriscalo** [comment]
I've used other macOS popover UIs that don't steal focus from the front window. Let me see if I can track down an example or two.

**Comment by iby** [comment]
@chriscalo That's true and it's the standard behaviour. However, there was a good reason for doing that, which I can't recall, but something to do with interactions. I can see other apps (1Password and Dropbox) with complex popover UI are not doing that, so maybe we should revisit this indeed. I'll post an update here.

**Comment by mrleblanc101** [comment]
Maybe you can keep stealing focus, but just ignore the first click in the render ?

**Comment by iby** [comment]
We want to try and fix this by avoiding stealing the focus in the first place – this is the preferred solution. Hopefully in one of the upcoming releases. In the meantime, what I also do sometimes is just Cmd + Tab while in selection mode – it works just fine to refocus on the app. But this is not a cure, just a personal tip… 🙂
